libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -636,6 +639,28 @@ public boolean isIDPLoginFlowEnabled() {
         return !TextUtils.isEmpty(idpAppURIScheme);
     }
 
+    /**
+     * Returns whether this application is configured as a Identity Provider
+     * @return True - if
+     */
+
+    public boolean isIdentityProvider() {
+        try {
+            PackageInfo packageInfo = context.getPackageManager().getPackageInfo(context.getPackageName(), PackageManager.GET_ACTIVITIES);
+            for (ActivityInfo activityInfo : packageInfo.activities) {
+                if (activityInfo.name.equals(IDPAccountPickerActivity.class.getName())) {
```

Comment:
Wouldn't this always be there anyway? Or does it depend on what's defined in the `Manifest` file?